src/main/java/uk/gov/hmcts/reform/hmc/model/hearing/PartyType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.hmc.model.hearing;
+
+public enum PartyType {
+    IND,
```

Comment:
I think INDIVIDUAL and ORGANIZATION are more readable in code

Reply:
This is existing code pulled over from civil-service repo into common place. not changing this here.